simple_salesforce/tests/test_api.py

```diff
@@ -705,7 +732,7 @@ def test_query_all(self):
             OrderedDict([(u'records', [
                 OrderedDict([(u'ID', u'1')]),
                 OrderedDict([(u'ID', u'2')])
-            ]), (u'done', True)]))
+            ]), (u'done', True), (u'totalSize', 2)]))
```

Comment:
Can remove "u" from OrderedDicts to be consistent with current test_api convention and removing support for python 2.*

Reply:
rebased on the current master and resolved this